vector/src/main/java/im/vector/app/features/settings/notifications/VectorSettingsNotificationPreferenceFragment.kt

```diff
@@ -145,6 +147,7 @@ class VectorSettingsNotificationPreferenceFragment @Inject constructor(
         refreshBackgroundSyncPrefs()
 
         handleSystemPreference()
+        handleVersionedSettings()
```

Comment:
Can we observe some blink with this new way to hide preference? Maybe hide both pref by default in the XML

Reply:
there's a blink if the visibility is updated in `onResume` but `onBindPref` is instant (as long as we stay on the main thread)

the programmatic visibility is also removed in #4627 :tada: